src/View/HTML.php

```diff
@@ -54,8 +64,10 @@ public static function createTag($tag, $attributes, $content = null)
         // Build list of arguments
         $preparedAttributes = '';
         foreach ($attributes as $attributeKey => $attributeValue) {
+            $whitelisted = in_array($attributeKey, static::config()->get('legal_empty_attributes'));
```

Comment:
Tiny optimisation - extract this config accessor into a variable before the `foreach` loop.